client/src/components/OverviewComponent/index.js

```diff
@@ -17,11 +16,12 @@ const OverviewComponent = ({
   isLeftImg,
   isHomePage,
   isDark,
+  isRed,
 }) => {
   return (
     <div className="overview-container">
       {isLeftImg && (
```

Comment:
instead of writing this two times
```jsx
{isLeftImg && (
        <div className={isRed ? 'redGround' : 'whiteGround'}>
          <img className="home-photo" src={imageSource} alt="Home" />
        </div>
      )}
{isRightImg && (
        <div className={isRed ? 'redGround' : 'whiteGround'}>
          <img className="home-photo" src={imageSource} alt="Home" />
        </div>
      )}
```
you can use CSS  order property to make it right or left and then you don't need `isLeftImg` props and don't need the second div,
you should make a condition in the first div like this
```jsx
<div
        className={`${isRightImg ? 'right' : ''} ${
          isRed ? 'redGround' : 'whiteGround'
        }`}
      >
        <img className="home-photo" src={imageSource} alt="Home" />
      </div>
```
and in css give the class right property order: 1